farmhelper-mod/src/main/java/com/jelly/farmhelper/features/ProfitCalculator.java

```diff
@@ -389,12 +389,18 @@ public static void fetchBazaarPrices() {
             for (BazaarItem item : cropsToCount) {
                 JSONObject json2 = (JSONObject) json1.get(item.bazaarId);
                 JSONObject json3 = (JSONObject) json2.get("quick_status");
-                bazaarPrices.put(item.localizedName, (Double) (json3).get("buyPrice"));
+                if ((bazaarPrices.get(item.localizedName) != null) || (bazaarPrices.get(item.localizedName) != null && ((Double) json3.get("buyPrice") < bazaarPrices.get(item.localizedName) * 3.0f)))
```

Comment:
First condition is pointless, it should be == null. Change it to
`if ((bazaarPrices.get(item.localizedName) == null || ((Double) json3.get("buyPrice") < bazaarPrices.get(item.localizedName) * 3.0f)))`